debezium-standalone-quarkus/pom.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.debezium</groupId>
+        <artifactId>debezium-parent</artifactId>
+        <version>1.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>debezium-standalone-quarkus</artifactId>
```

Comment:
Quarkus is just an implementation detail, it shouldn't be exposed on the surface. I'd suggest to name this simply _debezium-server_, also avoiding "standalone" which some might associate with reduced functionality.